packages/relay/src/lib/eth.ts

```diff
@@ -1735,7 +1749,7 @@ export class EthImpl implements Eth {
     this.logger.trace(`${requestIdPrefix} getTransactionReceipt(${hash})`);
 
     const cacheKey = `${constants.CACHE_KEY.ETH_GET_TRANSACTION_RECEIPT}_${hash}`;
-    const cachedResponse = this.cacheService.get(cacheKey, EthImpl.ethGetTransactionReceipt, requestIdPrefix);
+    const cachedResponse = false; //this.cacheService.get(cacheKey, EthImpl.ethGetTransactionReceipt, requestIdPrefix);
```

Comment:
Left over changes to be resolved

Reply:
I forgot to return this to how it was 😓 
it was there for testing purposes.
Thank you for noticing this.

Reply:
✅ Done